### PR TITLE
Add parser falcontcl_backend_rfm

### PR DESCRIPTION
**Pull request by wushiqinlou**
### All Pull Requests:

Check all that apply:

* [ ] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:
Add new parser for new Advisor rule

### CODE REVIEW

**Comment by wushiqinlou** [comment]
@xiangce  @JoySnow - please confirm

File: insights/parsers/falcontcl.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            self.backend = content[0].split(".")[0].split("=")[-1].strip()
+        else:
+            self.backend = "not set"
+
+
+@parser(Specs.falcontcl_rfm)
+class FalconctlRfm(CommandParser):
+    """
+    This parser reads the output of ``/opt/CrowdStrike/falconctl -g --rfm-state``,
+    return the back-end mode as a string.
```
Comment (JoySnow) [inline insights/parsers/falcontcl.py:53]: This usage of `back-end mode` seems for the command above `/opt/CrowdStrike/falconctl -g --backend`. We need to diff the docstrings here.

**Review by JoySnow — COMMENTED** [review]
@wushiqinlou The changes looks good to me, except this docstrings part.

File: insights/parsers/falcontcl.py (hunk 1 above)
Comment (wushiqinlou) [reply insights/parsers/falcontcl.py]: @JoySnow - Updated, please check

**Review by xiangce — APPROVED** [review]

**Pull request merged**